python/pyiceberg/table/metadata.py

```diff
@@ -303,7 +306,7 @@ def to_v2(self) -> "TableMetadataV2":
         metadata["format_version"] = 2
         return TableMetadataV2(**metadata)
 
-    format_version: Literal[1] = Field(alias="format-version")
+    format_version: Literal[1] = Field(alias="format-version", default=1)
```

Comment:
I don't think we can set a default here, it is missing, we should raise an exception.

Reply:
Ok, I set it here because there is the default in the v2